rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -6399,10 +6391,10 @@ void opcode_0223(const ScriptArguments& args, const ScriptCharacter character, c
     @arg vehicle Car/vehicle
     @arg arg2 
 */
-void opcode_0224(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptInt arg2) {
+void opcode_0224(const ScriptArguments& args, const ScriptVehicle vehicle, const ScriptFloat health) {
```

Comment:
Urgh? This suggests that the SCM documentation we used for the opcode was bad. Please review or even unit-test this.
If you can't do this, please add an issue that we should review the type of this argument.

Reply:
Yeah, not sure if is there any up to date proper documentation anywhere. Most popular sources like gtamodding.com or gtagmodding.com have mistakes.